stand/efi/loader/bootinfo.c

```diff
@@ -186,7 +186,7 @@ bi_load_efi_data(struct preloaded_file *kfp, bool exit_bs)
 	struct efi_map_header *efihdr;
 	bool do_vmap;
 
-#if defined(__amd64__) || defined(__aarch64__)
+#if defined(__amd64__) || defined(__aarch64__) || defined(__uefi64__)
```

Comment:
The __uefi64__ bothers me a bit. What does it really mean?

Reply:
Shouldn't __i386__ be defined in this case?  I think that will suffice if it is... If we need more nuance in the future, we can go there then...  But since i386 is on its way out of FreeBSD and the number of i386 platforms that will start using it is nil,  I don't think we'll ever need anything else.

Reply:
Yea that makes sense, it would be the same with \_\_i386__ and if support for i386 is getting dropped, we can safely use it.